wordle-web/pages/wordEditor.vue

```diff
@@ -0,0 +1,364 @@
+        </v-tooltip>
+      </v-col>
+      <v-col>
+        <v-radio-group v-model="commonRadio">
+          <v-radio
+            label="Both common/uncommon"
+            :value="CommonRadio.Both"></v-radio>
+          <v-radio label="Only common" :value="CommonRadio.Common"></v-radio>
+          <v-radio
+            label="Only uncommon"
+            :value="CommonRadio.Uncommon"></v-radio>
+        </v-radio-group>
+      </v-col>
+    </v-row>
+    <v-table class="table my-2 mx-auto w-75 opacity-50">
+      <thead>
+        <tr>
+          <th
+            class="text-center text-h8"
+            @click="
+              isWordOrderAscending = !isWordOrderAscending;
+              sortWords(words!);
+            ">
+            <strong>Word</strong
+            ><v-icon
+              :icon="
+                isWordOrderAscending ? 'mdi-chevron-up' : 'mdi-chevron-down'
+              "></v-icon>
+          </th>
+          <th
+            class="text-center text-h8"
+            @click="
+              commonWordOrder = (commonWordOrder + 1) % 3;
+              sortWords(words!);
+            ">
+            <strong>Common Word</strong
+            ><v-icon :icon="getCommonWordIcon()"></v-icon>
+          </th>
+          <th class="text-center text-h8"><strong>Edit</strong></th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr v-for="(word, index) in words">
+          <td class="text-center text-h8">
+            {{ word.word }}
+          </td>
+          <td class="text-center text-h8">
+            {{ word.isCommonWord ? 'Yes' : 'No' }}
+          </td>
+          <td class="text-center text-h8">
+            <v-row class="mt-0">
+              <v-col cols="3">
+                <v-tooltip
+                  text="Tooltip"
+                  :disabled="tokenService?.isLoggedIn()">
+                  <template v-slot:activator="{ props }">
+                    <div class="text-center mx-0" v-bind="props">
+                      <v-switch
+                        :color="tokenService?.isLoggedIn() ? 'primary' : ''"
+                        :disabled="!tokenService?.isLoggedIn()"
+                        v-model="word.isCommonWord"
+                        @update:modelValue="
+                          isCommonWordChanged[index] =
+                            !isCommonWordChanged[index]
+                        "></v-switch>
+                    </div>
+                  </template>
+                  <span>You must be logged in to change common word flag</span>
+                </v-tooltip>
+              </v-col>
+              <v-col class="mt-2" cols="4">
+                <v-tooltip
+                  text="Tooltip"
+                  :disabled="isOlderThanTwentyOne && isMotU">
+                  <template v-slot:activator="{ props }">
+                    <div class="text-center mx-0" v-bind="props">
+                      <v-btn
+                        :color="isOlderThanTwentyOne && isMotU ? 'error' : ''"

```

Comment:
You seem to be using isOlderThanTwentyOne && isMotU in tandem a lot. I would suggest creating a computed property for this called `CanAddOrDelete` and just use that instead since that's what having these two claims means.

Reply:
That's a great idea. Thank you :)